tests/integration/test_exodus.py

```diff
@@ -30,7 +30,6 @@ def test_header_not_exist_file(cdn_test_url):
 # Test data for exodus-lambda
 # serving repo entry points with appropriate cache headers
 testdata_cache_control_path = [
-    "/content/dist/rhel/server/5/5.7/listing",
```

Comment:
I don't think this should be removed.  I think a test failure here is accurately showing that we missed something:

If you have a look in the origin response function, you'll see that listing files are some of those where we intentionally set the cache-control header.  However since we now generate the listing responses in origin request, it means that origin response is no longer reached for listings.  So we've lost the cache-control header for listings.

The test has detected this bug, I think the appropriate response is to update the listing code in origin request to restore the lost header, and not silence the test.